alpha-core/src/main/java/at/ac/tuwien/kr/alpha/core/rules/InternalRule.java

```diff
@@ -143,4 +142,22 @@ public int getRuleId() {
 		return this.ruleId;
 	}
 
+	/**
+	 * Returns a copy of the rule with the specified literal removed, or null if the new rule would have an empty body.
+	 * @return the rule without the specified literal, null if no body remains.
+	 */
+	public InternalRule returnCopyWithoutLiteral(Literal literal) {
```

Comment:
Is there a particular reason to stress "return" here?
```suggestion
	public InternalRule copyWithoutLiteral(Literal literal) {
```